nav2_msgs/srv/SaveMap3D.srv

```diff
@@ -0,0 +1,11 @@
+# URL of map resource
+# Can be an absolute path to a file: file:///path/to/maps/floor1.yaml
+# Or, relative to a ROS package: package://my_ros_package/maps/floor2.yaml
+string map_topic
+string map_url
+string origin_topic
```

Comment:
Swap `map_url` and `origin_topic`, since the last one have a relation to (go in a pair with) `map_topic`